DNN Platform/Library/Security/Permissions/PermissionInfo.cs

```diff
@@ -1,50 +1,60 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
 namespace DotNetNuke.Security.Permissions
 {
     using System;
     using System.Data;
     using System.Xml.Serialization;
-
+
+    using DotNetNuke.Abstractions.Security.Permissions;
     using DotNetNuke.Common.Utilities;
-    using DotNetNuke.Entities;
-    using Newtonsoft.Json;
-
+    using DotNetNuke.Entities;
+    using Newtonsoft.Json;
+
     /// Project  : DotNetNuke
     /// Namespace: DotNetNuke.Security.Permissions
     /// Class    : PermissionInfo
     /// <summary>PermissionInfo provides the Entity Layer for Permissions.</summary>
     [Serializable]
-    public class PermissionInfo : BaseEntityInfo
+    public class PermissionInfo : BaseEntityInfo, IPermissionInfo
     {
-        /// <summary>Gets or sets the Mdoule Definition ID.</summary>
-        /// <returns>An Integer.</returns>
-        [XmlIgnore]
+        /// <inheritdoc cref="IPermissionInfo.ModuleDefId" />
+        [XmlIgnore]
         [JsonIgnore]
         public int ModuleDefID { get; set; }
```

Comment:
Do we want this to be deprecated?

Reply:
See my comment in https://github.com/dnnsoftware/Dnn.Platform/pull/5838#discussion_r1358508810, the same thing applies here as well.

Reply:
@dnnsoftware/approvers opinions?

Reply:
The discussion around this is to mark as [Obsolete] as needed